code/game/machinery/_machinery.dm

```diff
@@ -927,10 +927,10 @@
 	toggle_panel_open()
 	if(panel_open)
 		icon_state = icon_state_open
-		to_chat(user, span_notice("You open the maintenance hatch of [src]."))
```

Comment:
Keep this as to_chat() for now. Some machines(including canisters) don't have a visual icon to show that it's panel is open. Displaying a chat that sticks around helps remind players that they last opened the panel